webapp/packages/plugin-authentication/src/UserInfo.tsx

```diff
@@ -9,52 +9,53 @@
 import { observer } from 'mobx-react-lite';
 import styled, { css } from 'reshadow';
 
-import { topMenuStyles } from '@cloudbeaver/core-app';
-import { AuthInfoService, DATA_CONTEXT_USER } from '@cloudbeaver/core-authentication';
 import { IconOrImage } from '@cloudbeaver/core-blocks';
 import { useService } from '@cloudbeaver/core-di';
-import { useStyles } from '@cloudbeaver/core-theming';
-import { ContextMenu } from '@cloudbeaver/core-ui';
-import { useMenu } from '@cloudbeaver/core-view';
+import { useTranslate } from '@cloudbeaver/core-localization';
+import type { UserInfo as IUserInfo } from '@cloudbeaver/core-sdk';
+import { composes, useStyles } from '@cloudbeaver/core-theming';
+import { UserProfileService } from '@cloudbeaver/plugin-user-profile';
 
-import { MENU_USER_PROFILE } from './UserMenu/MENU_USER_PROFILE';
-import { userMenuStyles } from './UserMenu/userMenuStyles';
+const styles = composes(
+  css`
+    user {
+      composes: theme-ripple from global;
+    }
+  `,
+  css`
+    user {
+      height: 100%;
+      display: flex;
+      align-items: center;
+      padding: 0 16px;
+      cursor: pointer;
+    }
+    IconOrImage {
+      display: block;
+      width: 24px;
+    }
+    user-name {
+      display: block;
+      line-height: initial;
+      margin-left: 8px;
+    }
+`);
 
-const styles = css`
-  user {
-    height: 100%;
-    display: flex;
-    align-items: center;
-  }
-  IconOrImage {
-    display: block;
-    width: 24px;
-  }
-  user-name {
-    display: block;
-    line-height: initial;
-    margin-left: 8px;
-  }
-`;
+interface Props {
+  info: IUserInfo;
+}
 
-export const UserInfo = observer(function UserInfo() {
-  const authInfoService = useService(AuthInfoService);
-  const style = useStyles(styles, userMenuStyles);
-  const menu = useMenu(MENU_USER_PROFILE);
-  menu.context.set(DATA_CONTEXT_USER, authInfoService.userInfo);
-
-  if (!authInfoService.userInfo) {
-    return null;
-  }
+export const UserInfo = observer<Props>(function UserInfo({ info }) {
+  const translate = useTranslate();
+  const style = useStyles(styles);
+  const userProfileService = useService(UserProfileService);
```

Comment:
Wrong dependency direction. We need to move user info to `@cloudbeaver/plugin-user-profile`.